mass_machine_type_transition_job.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mass-machine-type-transition
+spec:
+  template:
+    metadata:
+      name: mass-machine-type-transition
+    spec:
+      containers:
+      - name: machine-type-transition
+        image: machine-type-transition:latest
+        imagePullPolicy: IfNotPresent
+      restartPolicy: OnFailure
```

Comment:
Please add the environment variables with default values to the job manifest. It will help guide users to use them properly.

```yaml
apiVersion: batch/v1
kind: Job
metadata:
  name: mass-machine-type-transition
spec:
  template:
    metadata:
      name: mass-machine-type-transition
    spec:
      containers:
      - name: machine-type-transition
        image: machine-type-transition:latest
        imagePullPolicy: IfNotPresent
        env:
        - name: NAMESPACE
          value: ""  # default is having no selector
        - name: RESTART_NOW
          value: false
      restartPolicy: OnFailure
```